ting_subsearch_suggestions.install

```diff
@@ -52,7 +57,17 @@ function ting_subsearch_suggestions_schema() {
 function ting_subsearch_suggestions_install() {
   // Install translations.
   $file = new stdClass();
-  $file->uri = drupal_get_path('module', 'ting_subsearch_opensuggestion') . '/translations/da.po';
+  $file->uri = drupal_get_path('module', 'ting_subsearch_suggestions') . '/translations/da.po';
   $file->filename = basename($file->uri);
   _locale_import_po($file, 'da', LOCALE_IMPORT_OVERWRITE, 'default');
 }
+
+/**
+ * Add "weekno" column in database.
+ */
+function ting_subsearch_suggestions_update_7001() {
+  db_drop_table('ting_subsearch_kpi_index_popular_keys');
```

Comment:
I assume we are certain that these values can be deleted without consequences.

Reply:
Yes, this data can be deleted because we are moving this functionality to support extended data format with new workflow.